hades/api.py

```diff
@@ -119,7 +119,9 @@ def users_api():
     table = get_table_by_name(table_name)
     if table is None:
         return jsonify({'message': f'Table {table_name} does not exist!'}), 400
-    return jsonify(users_to_json(table.query.all())), 200
+    if request.args.get('csv'):
```

Comment:
This works well - planning to call this via AJAX if the user wants (on registrations page) to be able to easily download a CSV